dags/airflow_db_cleanup.py

```diff
@@ -1,268 +0,0 @@
-"""
-A maintenance workflow that you can deploy into Airflow to periodically clean
```

Comment:
Don't need these anymore – the Airflow CLI comes with a handy cleanup script we can run.